dds/DCPS/RTPS/Spdp.cpp

```diff
@@ -1344,6 +1467,19 @@ Spdp::part_bit()
     bit_subscriber_->lookup_datareader(DCPS::BUILT_IN_PARTICIPANT_TOPIC);
   return dynamic_cast<DCPS::ParticipantBuiltinTopicDataDataReaderImpl*>(d.in());
 }
+
+DCPS::ParticipantLocationBuiltinTopicDataDataReaderImpl*
+Spdp::part_loc_bit()
```

Comment:
This appears to be in DiscoveryBase.h.  Can that one be used instead of re-defining it?  This may also be a problem with `part_bit`.